packages/grafana-ui/src/components/Table/TableNG/Filter/Filter.tsx

```diff
@@ -0,0 +1,81 @@
+import { css, cx } from '@emotion/css';
+import { useCallback, useMemo, useRef, useState } from 'react';
+
+import { Field, GrafanaTheme2, SelectableValue } from '@grafana/data';
+
+import { Popover } from '../../..';
+import { useStyles2 } from '../../../../themes';
+import { Icon } from '../../../Icon/Icon';
+
+import { REGEX_OPERATOR } from './FilterList';
+import { FilterPopup } from './FilterPopup';
+
+interface Props {
+  name: string;
+  rows: any[];
+  filter: any;
+  setFilter: (value: any) => void;
+  field?: Field;
+}
+
+export const Filter = ({ name, rows, filter, setFilter, field }: Props) => {
+  const filterValue = filter[name]?.filtered;
```

Comment:
```suggestion
  if (!field || !field.config.custom?.filterable) {
    return null;
  }
  const filterValue = filter[name]?.filtered;
```

Is there a reason this was removed? I noticed it was commented out in a previous commit. Without it, we can't do field overrides for disabling filter for a column.

Reply:
In fact, enabling a filter using `field.config.custom.filterable` is not supported as well.

Reply:
indeed, let me add it back